### ISSUE TITLE
Migracja z parcel-management na shipment-management, ubezpieczenie

### ISSUE THREAD

**Issue by Tomash667** [allegro/allegro-api#8969]
Szukałem w dokumentacji ale nie widzę żeby to było gdzieś jasno opisane.

W *starym* api (parcel-management) było pole deklarowana wartość na paczce.
W nowym api ( shipment-management) na paczce nic nie ma, za to można ustawić wartość "insurance".

Czy jest jakaś zależność pomiędzy tymi wartościami? Czy jak wcześniej ustawiałem deklarowaną wartość to teraz muszę tą wartość ustawiać jako ubezpieczenie czy to są zupełnie niepowiązane pola i wystarczy nic nie ustawiać?

Korzystam z Allegro One Box/Punkt/Kurier jeśli ma to jakieś znaczenie.

**Comment by Lukasz-Zurek** [comment]
> Czy jak wcześniej ustawiałem deklarowaną wartość to teraz muszę tą wartość ustawiać jako ubezpieczenie czy to są zupełnie niepowiązane pola i wystarczy nic nie ustawiać?

Tak. Tak na prawdę pole "insurance", które udostępniliśmy na nowych zasobach **/shipment-management**, zastąpiło niejako "items.value", z tym, że "insurance" bardziej czytelnie opisuje funkcjonalność i skupia się na ubezpieczeniu przesyłki. Na nowym zasobie nie udostępnialiśmy pola z /parcel-management. Nie ustawisz "wartości paczki". 

> Korzystam z Allegro One Box/Punkt/Kurier jeśli ma to jakieś znaczenie.

Nie ma to znaczenia, to funkcjonuje również dla tych przesyłek.
